templates/api.yaml

```diff
@@ -52,6 +53,10 @@ Parameters:
   JobTableName:
     Description: Table name for Jobs Table
     Type: String
+  LambdaAPIMemorySize:
+    Type: Number
+    MinValue: 128
+    MaxValue: 10240
```

Comment:
Nitpick: I don't think we need Min/Max validation here since it's done in the parent stack.

Reply:
Lol I did this because cfn-lint was erroring 😢